src/Service/Core.php

```diff
@@ -116,4 +116,64 @@ public function getConfigFromJson($module) {
     return $json;
   }
 
+  /**
+   * Get the current text and configuration settings for an app.
+   *
+   * @param string $module
+   *   The machine name of the extending module.
+   * @param string $language
+   *   The language code.
+   *
+   * @return string
+   *   The text and configuration settings for the app, as JSON.
+   *
+   * @throws \Drupal\Component\Plugin\Exception\InvalidPluginDefinitionException
+   * @throws \Drupal\Component\Plugin\Exception\PluginNotFoundException
+   */
+  public function getAppConfig($module, $language) {
+    // Get the relevant node for the app.
+    $node = $this->getAppNode($module, $language);
```

Comment:
Better to have check if $node is empty or not, also if there are more that one item.

Reply:
good call on checking empty

I think we'll need to prevent multiple nodes having the same app ID - see https://github.com/Capgemini/spalp/pull/8/files#diff-17726fb4e02d3c343d5291762cc76262R163